src/streamlink/plugins/afreeca.py

```diff
@@ -75,6 +70,10 @@ class AfreecaTV(Plugin):
                 validate.optional("CDN"): validate.any(str, None),
                 validate.optional("BJNICK"): validate.any(str, None),
                 validate.optional("TITLE"): validate.any(str, None),
+                validate.optional("VIEWPRESET"): [{
+                    "label": validate.transform(str),
+                    "name": validate.transform(str),
```

Comment:
Don't use `validate.transform(str)`, otherwise there's no type validation and a `None` value could be turned into a `str`, which would result in `"None"`. Just use `str` please.